backend/python_init_script/main.py

```diff
@@ -32,33 +32,34 @@ def load_user_data(file: str) -> [UserData]:
     :return: iterable of the loaded UserData objects
     :rtype: [UserData]
     """
+
+    def _replace_nan(val) -> str:
+        # if cell has no value in xlsx/csv it will result in a nan value
+        return "" if str(val) == "nan" else str(val)
```

Comment:
I also would have liked a function which hides the implementation detail of `nan` like so:
```suggestion
def _get_team(row) -> str:
    return (team := str(row["Team"])) if team != "nan" else ""
```

I think, the code makes clear, that `nan` appears when the string should be empty so that readers don't need to care about 'nan' when reading code in the caller function.

If you like it, you can apply this change.